conda/vs2022/activate.bat

```diff
@@ -0,0 +1,51 @@
+:: Set env vars that tell distutils to use the compiler that we put on path
+SET DISTUTILS_USE_SDK=1
+SET MSSdk=1
+
+SET "VS_VERSION=17.4"
+SET "VS_MAJOR=17"
+SET "VC_YEAR=2022"
+SET "VC_VERSION_LOWER=17"
+set "VC_VERSION_UPPER=18"
+
+set "MSYS2_ARG_CONV_EXCL=/AI;/AL;/OUT;/out"
+set "MSYS2_ENV_CONV_EXCL=CL"
```

Comment:
Why lowercase set here? Do we have some sort of linter
```suggestion
SET "VC_VERSION_UPPER=18"

SET "MSYS2_ARG_CONV_EXCL=/AI;/AL;/OUT;/out"
SET "MSYS2_ENV_CONV_EXCL=CL"
```

Reply:
I've basically copied what's in the vs2017 folder already. I can certainly unify the casing. So the upper-case is the code style adopted? Or rewrite to something else than Batch.

Reply:
My point is more that we should not keep copying those files over and over, but finally switch to PowerShell while doing the upgrade.

Reply:
Unless I am missing something, those Batch files are needed for the `conda-build` recipe and it does not seem to support PowerShell yet (conda/conda-build#3967). I've at least removed the recipe for VS2017 and unified the code style casing.